Firestore/Source/Model/FSTFieldValue.mm

```diff
@@ -748,7 +679,10 @@ - (id)value {
       HARD_FAIL("TODO(rsgowman): implement");
     case FieldValue::Type::String:
       return util::WrapNSString(self.internalValue.string_value());
-    case FieldValue::Type::Blob:
+    case FieldValue::Type::Blob: {
+      auto &blob = self.internalValue.blob_value();
```

Comment:
nit: The `blob` name doesn't seem useful to me, especially given the method that returns it is already named `blob_value()`. I'd either introduce the type (rather than auto) or just inline it with the return statement. Probably the latter. Optional.